llvm/utils/lit/lit/TestRunner.py

```diff
@@ -565,6 +566,18 @@ class SHFILEOPSTRUCTW(Structure):
     return ShellCommandResult(cmd, "", stderr.getvalue(), exitCode, False)
 
 
+def executeBuiltinUmask(cmd, shenv):
+    """executeBuiltinUmask - Change the current umask."""
+    if len(cmd.args) != 2:
+        raise InternalShellError(cmd, "'umask' supports only one argument")
```

Comment:
```suggestion
        raise InternalShellError(cmd, "'umask' requires exactly one argument")
```

Reply:
Consistency is good. I don't feel that strongly about it to request a separate comment-fix patch.